features/ios/ios_config.feature

```diff
@@ -0,0 +1,16 @@
+Feature: iOS Config
+
+  Background:
+    Given I clear the Bugsnag cache
+
+  Scenario: Telemetry
+    When I run the game in the "IosNativeException" state
+    And I wait for 2 seconds
+    And I clear any error dialogue
+    And On Mobile I relaunch the app
+    And I run the game in the "StartSDKDefault" state
+    And I wait to receive an error
+    And the error payload field "events.0.usage.config.maxBreadcrumbs" equals 50
+    And the error payload field "events.0.usage.callbacks.onSession" equals 1
```

Comment:
I'm missing the purpose of these new scenarios, which don't introduce new scenario classes.  Do they somehow demonstrate that the new config works, or is there not actually any underlying functionality in place at the moment - so we're just verifying that it doesn't crash?  (If the latter, are existing test not sufficient for that?).

Reply:
The usage telemetry data will only be present in android and cocoa events, hence a seperate scenario needed to check it exists

Reply:
Sure, but there's already scenarios that seem to do the same thing ("iOS native exception Smoke Test" and "Android JVM Background Thread Smoke Test".

Can't you just add the two "the error payload field blah is" steps to those existing scenarios, saving the additional runtime and potential network flakes?

Reply:
done